pkgs/by-name/go/gogup/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "gogup";
+  version = "0.27.3";
+
+  src = fetchFromGitHub {
+    owner = "nao1215";
+    repo = "gup";
+    rev = "v${version}";
+    hash = "sha256-8DtD22kvGez2iX0VqoZ1zSydcNYnDz3r698nXEwtoZE=";
+  };
+
+  vendorHash = "sha256-yqCmo33ihkaPK8iL5cnCIGbOLkdXjuIWLwtgAa+KB8Y=";
+  doCheck = false;
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = with lib; {
+    description = "gup - Update binaries installed by 'go install' with goroutines.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

must not end in period

Reply:
Also probably shouldn't start with `gup -`

Reply:
Changed it